### ISSUE TITLE
Debian: Cockpit-Machines and Cockpit-Podman missing in Bullseye Backports

### ISSUE THREAD

**Issue by ne20002** [cockpit-project/cockpit#16438]
Cockpit version: 254
OS: Debian Bullseye
Page: Machines, Podman

Hi

I see that @martinpitt accepted Cockpit 254 into Bullseye Backports .. thank you very much. I love Cockpit. But I wonder if newer versions of Cockpit-Machines and Cockpit-Podman will be available in Bullseye Backports as well? Thoese are still on 'old' versions in the main repository: Cockpit-Machines: 239, Cockpit-Podman: 28-1.

I also wonder if other cockpit application shall be available for Debian or if it is intedend that those are missing, e.g. Cockpit-SELinux is not available (I believe due to troubleshootd not available for Debian?)

Kind regards
Stefan

**Comment by martinpitt** [comment]
I already uploaded cockpit-machines to Debian backports a few days ago. But as it is a new package, it'll first need to be reviewed. See https://ftp-master.debian.org/backports-new.html

I'll upload a c-podman backport in a bit.

I tried to enable [cockpit-sosreport](https://ftp-master.debian.org/new/cockpit_251-2.html) several times already, but it's been stuck in binary NEW for a while. Indeed cockpit-selinux is not packaged for Debian as that requires SEtroubleshoot. The only remaining skipped package is kdump, which I mostly just didn't get around to so far, and there was no demand.
